doc/kazoo/cluster-guide.md

```diff
@@ -1,9 +1,277 @@
+```
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc1.z100.somedomain.com -d '{"_rev":3-b13d076f367df4d0c52b236e654b836c", "zone":"z100"}'
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc2.z100.somedomain.com -d '{"zone":"z100"}'
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc3.z100.somedomain.com -d '{"zone":"z100"}'
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc1.z100.somedomain.com -d '{"zone":"z200"}'
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc2.z100.somedomain.com -d '{"zone":"z200"}'
+curl -X PUT bc1.z100.somedomain.com:5986/nodes/bigcouch@bc3.z100.somedomain.com -d '{"zone":"z200"}'
+```
+Verify cluster:  
+`curl http://bc1.z100.somedomain.com:5984/_membership`  
+Should return:  
+```
+{"all_nodes":["bigcouch@bc3.z200.somedomain.com","bigcouch@bc2.z200.somedomain.com","bigcouch@bc1.z200.somedomain.com",
+"bigcouch@bc3.z100.somedomain.com","bigcouch@bc2.z100.somedomain.com","bigcouch@bc1.z100.somedomain.com"],
+
+"cluster_nodes":["bigcouch@bc3.z200.somedomain.com","bigcouch@bc2.z200.somedomain.com","bigcouch@bc1.z200.somedomain.com",
+"bigcouch@bc3.z100.somedomain.com"","bigcouch@bc2.z100.somedomain.com","bigcouch@bc1.z100.somedomain.com"]}
+```
+You can do that on each server to verify they all have the same configuration.
+
+To verify zone configuration on each document on each server.
+```
+curl http://bc1.z100.somedomain.com:5986/nodes/bigcouch@bc1.z100.somedomain.com
+curl http://bc1.z100.somedomain.com:5986/nodes/bigcouch@bc2.z100.somedomain.com
+...
+```
+
+etc.  
+### Kazoo Cluster
+
+After installing kazoo in both zones edit `/etc/kazoo/core/config.ini`.  This is exactly the same on both.  
+```
+; section are between [] = [section]
+; key = value
+; to comment add ";" in front of the line
+;[amqp]
+;uri = "amqp://guest:guest@127.0.0.1:5672"
+
+[zone]
+name = "z100"
+amqp_uri = "amqp://guest:guest@10.100.30.1"
+
+[zone]
+name = "z200"
+amqp_uri = "amqp://guest:guest@10.200.30.1"
+
+[bigcouch]
+compact_automatically = true
+cookie = change_me
+ip = "127.0.0.1"
+port = 15984
+; username = "kazoo"
+; password = "supermegaexcellenttelephonyplatform"
+admin_port = 15986
+
+[whistle_apps]
+cookie = change_me
+
+[kazoo_apps]
+cookie = change_me
+zone = "z100"
+host = "kz1.z100.somedomain.com"
+
+[kazoo_apps]
+cookie = change_me
+zone = "z200"
+host = "kz1.z200.somedomain.com"
+
+[ecallmgr]
+cookie = change_me
+zone = "z100"
+host = "kz1.z100.somedomain.com"
+
+[ecallmgr]
+cookie = change_me
+zone = "z200"
+host = "kz1.z200.somedomain.com"
+
+[log]
+syslog = info
+console = notice
+file = error
+```
+
+### Assign Freeswitch to Ecallmgr zones
+Edit the database using Futon or Fauxton by browsing to the following link.  
+http://bc1.z100.somedomain.com:5984/_utils/document.html?system_config/ecallmgr
+
+Add the following to the root of the document.  So at the same level as `"default":`
+```
+"z100": {
```

Comment:
I think I figured it out.
`sup ecallmgr_maintenance add_fs_node freeswitch@freeswitch1.z100.somedomain.com 'false'`

That adds it as ecallmgr node specific in the DB.